hd/etc/perso_module/relations.txt

```diff
@@ -55,191 +55,213 @@
+    <ul>
+    %( parrains, parents adoptifs, ... %)
+    %foreach;relation;
+      %if;(has_relation_him and has_relation_her)
+        <li>%apply;capitalize(relation_type)[:]%nl;
+          <ul>
+            %apply;li_SDC("relation_him")
+              %apply;image_MF("relation_him")
+              %apply;short_display_person("relation_him")
+            </li>
+            %apply;li_SDC("relation_her")
+              %apply;image_MF("relation_her")
+              %apply;short_display_person("relation_her")
+            </li>
+          </ul>
+        </li>
+      %elseif;has_relation_him;
+        %apply;li_SDC("relation_him")%apply;capitalize(relation_type)[:]%sp;
+          %apply;image_MF("relation_him")
+          %apply;short_display_person("relation_him")
+        </li>
+      %elseif;has_relation_her;
+        %apply;li_SDC("relation_her")%apply;capitalize(relation_type)[:]%sp;
+          %apply;image_MF("relation_her")
+          %apply;short_display_person("relation_her")
+        </li>
+      %end;
     %end;
-  %end;
-  %let;g_cnt;%count;%in;
-  %if;(g_cnt>0)
-    <li>%if;(sex=0)[*godfather/godmother/godparents]0
-         %elseif;(sex=1)[*godfather/godmother/godparents]1
-         %else;[*godparent]%end; [of]%sp;
-    %reset_count;
+    %( filleuls, enfants adoptés, ...%)
     %foreach;related;
-      %if;(related_type=[godson/goddaughter/godchild]0 or
-         related_type=[godson/goddaughter/godchild]1 or
-         related_type=[godson/goddaughter/godchild]2)
+      %apply;li_SDC("related")%apply;capitalize(related_type)[:]%sp;
+        %apply;image_MF("related")
         %apply;short_display_person("related")
-        %incr_count;
-        %if;(count=g_cnt).
-        %else;
-          %if;(count=g_cnt-1)%sp;[and]%sp;%else;,%sp;%end;
+      </li>
+    %end;
+    %( moi, témoin d'un événement %)
+      %foreach;event_witness_relation;
+        %apply;li_SDC("event_witness_relation.person")
+          %if;(event_witness_relation.event.date.year != "")
+            %apply;a_to_b%with;
+              %apply;capitalize(event_witness_relation_kind)
+              %sp;[in (month year)]%sp;%event_witness_relation.event.date.year;
+            %and;%event_witness_relation.event.name;%end;
+          %else;
+            %apply;a_to_b%with;
+            %apply;capitalize(event_witness_relation_kind)%and;
+            %event_witness_relation.event.name;%end;
+          %end;
+          %if;(event_witness_relation.person.index!=central_index)
+            %sp;%apply;a_of_b%with;%and;%apply;short_display_person("event_witness_relation.person")%end;
+            %if;(event_witness_relation.event.spouse != "")
+              %sp;[and]
+              %apply;short_display_person("event_witness_relation.event.spouse")
+            %end;
+          %end;.
+        </li>
+      %end;
+    </ul>
+  
+  %elseif;(op_m=2 or op_m=3)
+    %if(count1>0)
+      %if;(op_m=3)<h5>
+      [
+en: Relations and events of which he or her has been witness
```

Comment:
Please stop doing that. Create a new key word in lex_utf8. Or you won't be able to keep a good tracking of un-translated sentences.